Resources/Computing/SSHComputingElement.py

```diff
@@ -602,7 +602,14 @@ def _getJobOutputFiles( self, jobID, host = None ):
     if 'OutputTemplate' in self.ceParameters:
       self.outputTemplate = self.ceParameters['OutputTemplate']
       self.errorTemplate = self.ceParameters['ErrorTemplate']
-    
+
+    if self.batchSystem == 'Torque':
+      output = self.outputTemplate % jobStamp.split('.')[0]
+      error = self.errorTemplate % jobStamp.split('.')[0]
+    else:
+      output = self.outputTemplate % jobStamp
+      error = self.errorTemplate % jobStamp
```

Comment:
This fix is Torque dependent in a generic class. This is not good. If you just leave only two lines:
  output = self.outputTemplate % jobStamp.split('.')[0]
  error = self.errorTemplate % jobStamp.split('.')[0]
the result will be the same as before for "non-Torque" sites and correct for "Torque" sites.
However, I would like to ask you to rather modify the BatchSystems/Torque.py module so that the host name is stripped of in the submitJob method before returning the resulting JobIDs. In this case, the Torque pecularities will be contained in the specific Torque module. Since you have the case of such site you can make a proper test.